brd-ios/breadwallet/src/Redesign/Scenes/Swap/Details/SwapDetailsModels.swift

```diff
@@ -25,4 +25,10 @@ enum SwapDetailsModels {
         var header: AccessoryType? { return nil }
         var footer: AccessoryType? { return nil }
     }
+    
+    struct CopyValue {
+        struct ViewAction {
+            var value: String?
```

Comment:
no "XX copeied" snackbar uppon copying?

Reply:
It was not working anyway because the screen is presented without Coordinators from the Wallet.

Reply:
![image](https://user-images.githubusercontent.com/99484553/187368437-aeffd41e-3d01-48e9-b47f-4a85f1ea402e.png)